Update notes for Pull mode cluster in AA document (#1810)
karmada-io/karmada: docs/userguide/aggregated-api-endpoint.md

kubectl --kubeconfig karmada-apiserver.config get node
```

> > Note: For a member cluster that joins karmada in pull mode, if it uses a one-way network, we can [deploy apiserver-network-proxy (ANP)](../working-with-anp.md) to access it.

Reviewer: `one-way network` means that only member clusters can access the karmada control-plane, am I right.

Is this word well known? Do we need to explain this?

Author: > one-way network means that only member clusters can access the karmada control-plane, am I right?

Yes.

> > Note: For a member cluster that joins karmada in pull mode, if it uses a one-way network, we can [deploy apiserver-network-proxy (ANP)](../working-with-anp.md) to access it.

Reviewer 2: `if it uses a one-way network,`

I think we should give more clear comments here. 
cc @Tingtal for help
Here want to say is `maybe the cluster is behind a NAT and Karmada control plane can't access it directly.

Reviewer 3: I don't think there is a term to specifically describe such a situation... 
For me, I would say,
```suggestion
> Note: A member cluster that joins karmada in pull mode allows only cluster-to-karmada access. In this case, we can [deploy apiserver-network-proxy (ANP)](../working-with-anp.md) to access the cluster.
```

But here's another doubt. You said "**maybe** the cluster is behind a NAT". This means that there are other reasons for such one-way communication and **not all** member clusters in pull mode deny the access requests from the Karmada control plane, right?
If that's the case, I would say,

```suggestion
> Note: For a member cluster that joins karmada in pull mode and allows only cluster-to-karmada access, we can [deploy apiserver-network-proxy (ANP)](../working-with-anp.md) to access it.
```

Pls check my edits. Cheers!

Reviewer: Thanks!

Author: >  You said "maybe the cluster is behind a NAT". This means that there are other reasons for such one-way communication and not all member clusters in pull mode deny the access requests from the Karmada control plane, right?

Yes, thanks!

Reviewer 2: >  Note: For a member cluster that joins karmada in pull mode and allows only cluster-to-karmada access, we can [deploy apiserver-network-proxy (ANP)](../working-with-anp.md) to access it.

+1
